<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">* {	
	font-family:微软雅黑;
	font-size:15px;
	color: black;
}</string>
       </property>
       <property name="title">
        <string>拼音输入法</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
              <item>
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">* {	
	font-family:微软雅黑;
	font-size:15px;
	color: black;
}</string>
                </property>
                <property name="text">
                 <string>InPut</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="styleSheet">
                 <string notr="true">* {	
	font-family:微软雅黑;
	font-size:15px;
	color: #1d649c;
}</string>
                </property>
                <property name="placeholderText">
                 <string>请输入拼音...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,7">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="styleSheet">
                 <string notr="true">* {	
	font-family:微软雅黑;
	font-size:15px;
	color: black;
}</string>
                </property>
                <property name="text">
                 <string>OutPut</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextBrowser" name="textBrowser">
                <property name="styleSheet">
                 <string notr="true">* {	
	font-family:微软雅黑;
	font-size:25px;
	color: #1d649c;
}</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">* {	
	font-family:微软雅黑;
	font-size:15px;
	color: black;
}* {	
	font-family:微软雅黑;
	font-size:15px;
	color: black;
}</string>
              </property>
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit_2">
              <property name="styleSheet">
               <string notr="true">* {	
	font-family:微软雅黑;
	font-size:25px;
	color: #1d649c;
}</string>
              </property>
              <property name="placeholderText">
               <string>请输入要选择内容的序号...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
